### PR TITLE
Description about official support

### PR DESCRIPTION
**Pull request by mi-ac**
This description should serve as explanation that can easily be posted on the bugs filed using unsupported configurations.

### CODE REVIEW

File: src/docs/official-support.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+* Breakages on roughly the same [set of builders](https://chromium.googlesource.com/infra/infra/+/main/infra/services/lkgr_finder/config/v8_cfg.pyl) block our continuous roll into Chromium.
+* Some architecture ports are [handled externally](/docs/ports).
+* Some configurations are [experimental](https://ci.chromium.org/p/v8/g/experiments/console). Breakages are permitted and will be handled by the owners of the configuration.
+
+If you have a configuration that exhibits a problem, but is not covered by one of the bots above:
+* Feel free to submit a CL that fixes your problem. The team will support you with a code review.
+* You can use v8-dev@goolgegroups.com to discuss the problem.
+* If you think we should support this configuration (maybe a hole in our test matrix?), please file a bug on the [V8 issue tracker](https://bugs.chromium.org/p/v8/issues/entry) and ask.
+
+However: We don't have the bandwidth to support every possible configuration.
```
Comment (mathiasbynens) [inline src/docs/official-support.md:20]: To fix the lint issues:

```suggestion
Some nuances:

- Breakages on the most important builders will block code submission. A tree sheriff will usually revert the culprit.
- Breakages on roughly the same [set of builders](https://chromium.googlesource.com/infra/infra/+/main/infra/services/lkgr_finder/config/v8_cfg.pyl) block our continuous roll into Chromium.
- Some architecture ports are [handled externally](/docs/ports).
- Some configurations are [experimental](https://ci.chromium.org/p/v8/g/experiments/console). Breakages are permitted and will be handled by the owners of the configuration.

If you have a configuration that exhibits a problem, but is not covered by one of the bots above:

- Feel free to submit a CL that fixes your problem. The team will support you with a code review.
- You can use v8-dev@goolgegroups.com to discuss the problem.
- If you think we should support this configuration (maybe a hole in our test matrix?), please file a bug on the [V8 issue tracker](https://bugs.chromium.org/p/v8/issues/entry) and ask.

However, we don’t have the bandwidth to support every possible configuration.
```

**Review by mathiasbynens — CHANGES_REQUESTED** [review]

File: src/docs/official-support.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+- Some architecture ports are [handled externally](/docs/ports).
+- Some configurations are [experimental](https://ci.chromium.org/p/v8/g/experiments/console). Breakages are permitted and will be handled by the owners of the configuration.
+
+If you have a configuration that exhibits a problem, but is not covered by one of the bots above:
+
+- Feel free to submit a CL that fixes your problem. The team will support you with a code review.
+- You can use v8-dev@goolgegroups.com to discuss the problem.
+- If you think we should support this configuration (maybe a hole in our test matrix?), please file a bug on the [V8 issue tracker](https://bugs.chromium.org/p/v8/issues/entry) and ask.
+
+However: We don't have the bandwidth to support every possible configuration.
```
Comment (mathiasbynens) [inline src/docs/official-support.md:22]:
```suggestion
- You can use v8-dev@googlegroups.com to discuss the problem.
- If you think we should support this configuration (maybe a hole in our test matrix?), please file a bug on the [V8 issue tracker](https://bugs.chromium.org/p/v8/issues/entry) and ask.

However, we don’t have the bandwidth to support every possible configuration.
```

**Review by mathiasbynens — APPROVED** [review]
LGTM % suggestion

File: src/docs/index.md [hunk 3]
```diff
@@ -37,6 +37,7 @@ V8 enables any C++ application to expose its own objects and functions to JavaSc
     - [Checklist for staging and shipping of WebAssembly features](/docs/wasm-shipping-checklist)
     - [Flake bisect](/docs/flake-bisect)
     - [Handling of ports](/docs/ports)
+    - [Official support](/docs/official-support)
```
Comment (mathiasbynens) [inline src/docs/index.md:40]: Please maintain alpha-sorting

Comment (mi-ac) [reply src/docs/index.md:40]: Hmm, this list doesn't seem alpha sorted at all. Maybe only the 3-4 items around it here. Where should I move it?

I thought from a logical perspective it should stay side-by-side with the port handling. It's the closest in meaning.

Comment (mathiasbynens) [reply src/docs/index.md:40]: Ok, nevermind then

**Pull request merged**